packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/question..st

```diff
@@ -0,0 +1,14 @@
+as yet unclassified
+question: aQuestion
+
+	question := aQuestion.
+	selectedChoices := Set new.
+	(question type = 'singleChoice')
+	ifTrue: [self maxNumberOfChoices: 1.
+				self answer: LiquidSingleChoiceAnswer new.]
+	ifFalse: [(question type = 'multipleChoice')
+		ifTrue: [self maxNumberOfChoices: (question maxNumberOfChoices).
+					self answer: LiquidMultipleChoiceAnswer new.]
+		ifFalse: [self halt.]
```

Comment:
is there a reason why we halt? Wouldn't it be better to throw an error?

Reply:
@Dassderdie Was there a particular reason we chose halt over an error?

Reply:
The reasoning was basically:
"Here we should throw an error - but that's not relevant now, let's just put in a halt for now."

Reply:
Okay. We can combine that change with any feedback we get from Simon